lib/tasks/manageiq/cypress.rake

```diff
@@ -0,0 +1,69 @@
+namespace :cypress do
+  desc "Run cypress tests"
+  task :ui => 'ui:run'
+
+  namespace :ui do
+    desc "Seed database/configure assets for cypress specs"
+    task :seed do |rake_task|
+      app_prefix = rake_task.name.chomp('cypress:ui:seed')
+      Rake::Task["#{app_prefix}integration:seed"].invoke
+    end
+
+    desc "Interactively run cypress tests"
+    task :open => ['ui:setup'] do
+      sh "#{yarn_cmd} cypress open"
+    end
+
+    desc "Run headless cypress tests"
+    task :run => ['ui:setup'] do
+      sh "#{yarn_cmd} cypress run --headless --browser firefox --config video=false"
```

Comment:
this assumes everybody uses firefox, and doesn't run the `cypress:run:*` tasks  we already have

Reply:
(note we're using headless chrome on travis for spec:javascript, so it's already preinstalled)

Reply:
Yeah, sorry, this was for testing purposes for what I had on my machine, so this doesn't have to stay as is.

That said, the `video=false` flag doesn't exist on those tasks, which is why I just hard coded it to start and didn't just use the existing tasks.  I don't know if `yarn cypress:run:chrome` can take extra args (i.e. `yarn cypress:run:chrome --config video=false`), or if we have to create a new task.

We probably can add a task `yarn cypress:open:ci` which just uses chrome and also includes the `video=false` config.

Reply:
Aaaah

> Cypress records a video for each spec file when running tests during cypress run. Videos are not automatically recorded during cypress open.

I thought it never did that by default.

Yeah, I wonder if anyone uses video output from cypress:run, maybe we can just make video=false the default?

Or the task thing if you prefer :)